prebid-server/features/pbs-currency.md

```diff
@@ -80,7 +80,7 @@ Here are a couple examples showing the logic behind the currency converter:
 
 ## Request-Defined Conversion Rates
 
-Using PBS-Java, rates can be passed in on the request:
+With both PBS-Go and PBS-Java, custom currency conversion rates can be passed in the request:
```

Comment:
We don't need to differentiate if both versions support it. Consider more simply:

Rates can be passed in on the request:

Reply:
Modified